myokit/formats/sbml/_importer.py

```diff
@@ -35,334 +36,1065 @@ def __init__(self):
         self.re_name = re.compile(r'^[a-zA-Z]+[a-zA-Z0-9_]*$')
         self.re_alpha = re.compile(r'[\W]+')
         self.re_white = re.compile(r'[ \t\f\n\r]+')
-        self.units = {}
 
-    def _convert_name(self, name):
+    def info(self):
+        return info
+
+    def supports_model(self):
         """
-        Converts a name to something acceptable to myokit.
+        Returns a bool indicating if model import is supported.
         """
-        if not self.re_name.match(name):
-            org_name = name
-            name = self.re_white.sub('_', name)
-            name = self.re_alpha.sub('_', name)
-            if not self.re_name.match(name):
-                name = 'x_' + name
-            self.logger().warn(
-                'Converting name <' + org_name + '> to <' + name + '>.')
-        return name
+        return True
 
-    def _convert_unit(self, unit):
+    def supports_component(self):
```

Comment:
This should only return `True` if you've implemented a `component()` method that adds a component to an existing myokit model by importing it from a file.

Reply:
Oh I see! This be may actually be useful for the web app, but maybe better to do in another PR :D